lib/glimesh_web/live/homepage_live.ex

```diff
@@ -128,7 +128,7 @@ defmodule GlimeshWeb.HomepageLive do
 
                   <div class="card-stream-tags">
                     <%= if channel.subcategory do %>
-                      <span class="badge badge-info"><%= channel.subcategory.name %></span>
+                      <span class="badge "><%= channel.subcategory.name %></span>
```

Comment:
Does this make homepage tags look funky without the blue background?

Reply:
Im not entirely convinced i meant to do this change in all honesty. will take a look at it properly later today!